Add proposal for SciPy2022 (#7)
scientific-python/scientific-python-archive: presentations/proposed/scipy2022/general/proposal.md

https://www.scipy2022.scipy.org/talk-poster-presentations

## Scientific Python - From GitHub to TikTok

## Short Summary

The Scientific Python project aims to better coordinate the ecosystem and prepare Scientific Python for the next decade. We are a group of volunteers behind several projects from the ecosystem committed to expanding our community by generating a welcoming and friendly environment where projects collaborate and build together.

## Abstract

> As developers, sometimes we focus too much on writing code and too little in community. However, open source is about more than that. It is about communicating, teaching and collaborating. The Scientific Python project’s goal is to unify the community by promoting integration, diversity and wide participation, while generating resources that help bring contributors, developers and users together. 

Reviewer: ```suggestion
As developers, sometimes we focus too much on writing code and too little on community. However, open source is about more than that. It is about communicating, teaching, and collaborating. The Scientific Python project’s goal is to unify the community by promoting integration, diversity, and wide participation, while generating resources that help bring contributors, developers, and users together. 
```

### Documentation

> Documentation is a key aspect of every Python software project. Thus, we want to have a central landing place for information that will help users and contributors better understand the ecosystem. The scientific-python.org website contains resources that we hope are adopted by most of the projects. They include videos that orient new people to the ecosystem, interviews that introduce some of the members of our community and how they contribute to it, and demos, which give an overview of the problem solving workflow in the Python ecosystem. 

Reviewer 2: ```suggestion
Documentation is a key aspect of every Python software project. Thus, we want to have a central landing place for information that will help users and contributors better understand the ecosystem. The scientific-python.org website contains resources that we hope are adopted by most of the projects. They include videos that orient new people to the ecosystem, interviews that introduce some of the members of our community and how they contribute, and demos, which give an overview of the problem solving workflow in the Python ecosystem. 
```

### Blog 

The [Scientific Python blog](http://blog.scientific-python.org/) is a place where contributors can share their thoughts and ideas in order to achieve collaborative learning. We strongly believe that healthy communities are built when everyone’s voice is heard and different perspectives are valued.

### Social Media

> Community outreach and engagement are also part of growing our community. Nowadays, Facebook, Twitter, Instagram and TikTok are a big part of people's lives and as time passes by, communications change and people learn and collaborate differently. In order to stay connected, we will use social media to share our content and interact directly with users and developers. This will help us diversify our community and stay in touch with more people that are interested in contributing.

Reviewer: ```suggestion
Outreach and engagement are also important ways of growing our community. Nowadays, Facebook, Twitter, Instagram, and TikTok are a big part of people's lives and as time passes by, communications change and people learn and collaborate differently. In order to stay connected, we will use social media to share our content and interact directly with users and developers. This will help us diversify our community and stay in touch with more people that are interested in contributing.
```

I'm not quite sure I follow the point you are trying to make with "as time passes by, communications change and people learn and collaborate differently".

Reviewer 3: I would just delete the phrase "as time passes by, communications change and people learn and collaborate differently".

> Community outreach and engagement are also part of growing our community. Nowadays, Facebook, Twitter, Instagram and TikTok are a big part of people's lives and as time passes by, communications change and people learn and collaborate differently. In order to stay connected, we will use social media to share our content and interact directly with users and developers. This will help us diversify our community and stay in touch with more people that are interested in contributing.

Reviewer 3: ```suggestion
Community outreach and engagement are also part of growing our community. Nowadays, Facebook, Twitter, Instagram and TikTok are a big part of people's lives. In order to stay connected, we will use social media to share our content and interact directly with users and developers. This will help us stay in touch with more people that are interested in contributing.
```

### Come join us!

> This talk is an effort to share our view of the future Scientific Python community and a call to action for people to join us in this journey. We are in the search for more contributors for Scientific Python projects with different skills, backgrounds and perspectives to build an inclusive, diverse and friendly ecosystem.

Reviewer: ```suggestion
This talk shares our view of the future of the Scientific Python community and is a call to action for others to join us in this journey. We are looking for more contributors to Scientific Python projects with varying skills, backgrounds, and perspectives to build an inclusive, diverse and friendly ecosystem.
```
